osbs/api.py

```diff
@@ -966,3 +966,21 @@ def get_resource_quota(self, quota_name):
     @osbsapi
     def can_orchestrate(self):
         return self.build_conf.get_can_orchestrate()
+
+    @osbsapi
+    def create_config_map(self, config_name=None, config_data=None):
+        if not config_name or not config_data:
+            raise OsbsValidationException("Can't create a ConfigMap with no name or data")
```

Comment:
Just declare the method as `def create_config_map(self, config_name, config_data):`, since neither can be omitted.